### PR TITLE
[AMSFT102020]issaGruber

### PR DESCRIPTION
**Pull request by GruberI**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "node ./bin/www",
     "dev": "DEBUG=starter-code:* nodemon ./bin/www",
-    "dev-windows": "nodemon ./bin/www"
+    "dev-windows": "nodemon ./bin/www",
+    "seeds": "nodemon .bin/seed"
```
Comment (wax-bear) [inline package.json:9]: Nice!

File: public/stylesheets/style.scss [hunk 2]
```diff
@@ -6,3 +6,9 @@ body {
 a {
   color: #00B7FF;
 }
+
+.indexH1{
```
Comment (wax-bear) [inline public/stylesheets/style.scss:10]: This class does not seem to be used when rendering the index page.

**Review by wax-bear — COMMENTED** [review]
Great stuff Issa!
I placed only one minor comment regarding an unused class.

**Comment by GruberI** [comment]
Thank you Thomas!

**Pull request closed without merging**